Build/AdditionalProjectTransforms.xslt

```diff
@@ -0,0 +1,5 @@
+<xsl:if test="/Input/Properties/DefineNET4DetectionConstant = 'True'">
+  <PropertyGroup>
+    <DefineConstants Condition=" '$(TargetFrameworkVersion)' != 'v4.0' And '$(TargetFrameworkVersion)' != 'v3.5' And '$(TargetFrameworkVersion)' != 'v3.0' And '$(TargetFrameworkVersion)' != 'v2.0' ">$(DefineConstants);NET45</DefineConstants>
+  </PropertyGroup>
```

Comment:
As far as I have looked into this, if you want a compatibility with XBuild, no.

XBuild can't use semantics like ```$([System.Version]::Parse('$(TargetFrameworkVersion.Substring(1))').CompareTo($([System.Version]::Parse('4.5'))))``` which would have been totally generic.

But since MonoGame doesn't build on .Net 3.5 and below, we could simplify this condition to just be ```'$(TargetFrameworkVersion)' != 'v4.0'```.